src/iqm/qiskit_iqm/move_gate.py

```diff
@@ -60,3 +65,5 @@ def _define(self):
             qc._append(instr, qargs, cargs)
```

Comment:
I propose to change these lines 63-65 by removing them and defining the gate directly with its unitary:

```suggestion
        qc.unitary(self.unitary, [q[0], q[1]], label="move")
```

Reply:
I agree.